tests/utils/utils.test.jsx

```diff
@@ -294,3 +294,76 @@ describe('Utils.isValidPassword', function() {
         }
     });
 });
+
+describe('Utils.isEmail', function() {
+    test('', function() {
+        for (const data of [
+            {
+                email: 'prettyandsimple@example.com',
+                valid: true
+            },
+            {
+                email: 'very.common@example.com',
+                valid: true
+            },
+            {
+                email: 'disposable.style.email.with+symbol@example.com',
+                valid: true
+            },
+            {
+                email: 'other.email-with-dash@example.com',
+                valid: true
+            },
+            {
+                email: 'fully-qualified-domain@example.com',
+                valid: true
+            },
+            {
+                email: 'user.name+tag+sorting@example.com',
+                valid: true
+            },
+            {
+                email: 'x@example.com',
+                valid: true
+            },
+            {
+                email: 'example-indeed@strange-example.com',
+                valid: true
+            },
+            {
+                email: 'admin@mailserver1',
+                valid: true
+            },
+            {
+                email: '#!$%&\'*+-/=?^_`{}|~@example.org',
+                valid: true
+            },
+            {
+                email: 'example@s.solutions',
+                valid: true
+            },
+            {
+                email: 'Abc.example.com',
+                valid: false
+            },
+            {
+                email: 'A@b@c@example.com',
+                valid: false
+            },
+            {
+                email: '<Jonathan Fritz> jonathan.fritz@mattermost.com',
+                valid: false
+            },
+            {
+                email: 'comma@domain.com, separated@domain.com',
+                valid: false
+            },
+            {
+                email: 'comma@domain.com,separated@domain.com',
+                valid: false
+            }
+        ]) {
+            expect(Utils.isEmail(data.email)).toEqual(data.valid);
+        }
+    });
+});
```

Comment:
LGTM, just minor request:
Could you please add test case similar to `test <test@address.do>` which is based from the original report?

Reply:
Added in https://github.com/mattermost/mattermost-webapp/pull/491/commits/7a7d75854fb9bf4fb32e941293ca7e63a63f7307